testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -21,62 +21,65 @@
 from scipy.integrate import simps
 
 from numpy.testing import (assert_almost_equal, assert_equal,
-                           assert_array_almost_equal,raises)
+                           assert_array_almost_equal, raises)
 
 
-from MDAnalysisTests.datafiles import PDB, XTC
-
+from MDAnalysisTests.datafiles import PDB, XTC, RANDOM_WALK
 
 class TestPCA(object):
     def setUp(self):
         self.u = MDAnalysis.Universe(PDB, XTC)
         self.pca = pca.PCA(self.u.atoms, select='backbone and name CA',
-                           align=True)
+                           align=False)
         self.pca.run()
         self.n_atoms = self.u.select_atoms('backbone and name CA').n_atoms
 
     def test_cov(self):
         atoms = self.u.select_atoms('backbone and name CA')
-        reference = atoms.positions
-
-        ref_cog = atoms.center_of_geometry()
-        reference -= ref_cog
         xyz = np.zeros((10, 642))
         for i, ts in enumerate(self.u.trajectory):
-            mobile_cog = atoms.center_of_geometry()
-            mobile_atoms, old_rmsd = _fit_to(atoms.positions,
-                                             reference,
-                                             mobile_atoms=atoms,
-                                             mobile_com=mobile_cog,
-                                             ref_com=ref_cog)
-            xyz[i] = mobile_atoms.positions.ravel()
+            xyz[i] = atoms.positions.ravel()
 
         cov = np.cov(xyz, rowvar=0)
         assert_array_almost_equal(self.pca.cov, cov, 4)
 
     def test_cum_var(self):
         assert_almost_equal(self.pca.cumulated_variance[-1], 1)
-        # makes no sense to test values here, no physical meaning
 
     def test_pcs(self):
-        assert_equal(self.pca.p_components.shape, (self.n_atoms*3, self.n_atoms*3))
+        assert_equal(self.pca.p_components.shape,
```

Comment:
I'm thinking of just testing for some values of the variance and a vector or two. You would calculate it as the reference with the current version. That we can use as a regression test.